### PR TITLE
Compute resource config

### PR DESCRIPTION
**Pull request by magland**

### CODE REVIEW

File: python/api/api.py [hunk 1]
```diff
@@ -32,36 +32,13 @@
 
 def main():
     config_path_or_url = os.environ.get('LABBOX_EPHYS_CONFIG', None)
+    if config_path_or_url is None:
+        default_config_path = f'{thisdir}/../../labbox_config.yml'
```
Comment (magland) [inline python/api/api.py:36]: Look for the labbox config in a default location

File: python/labbox_ephys/api/_workersession.py [hunk 2]
```diff
@@ -49,11 +49,12 @@ def initialize(self):
 
         server_info = {
             'nodeId': node_id,
-            'defaultFeedId': self._default_feed_id
+            'defaultFeedId': self._default_feed_id,
+            'labboxConfig': self._labbox_config
```
Comment (magland) [inline python/labbox_ephys/api/_workersession.py:53]: Send the labboxConfig info to the browser

File: src/containers/Config.tsx [hunk 3]
```diff
@@ -38,14 +38,14 @@ const Config: FunctionComponent<Props> = ({ extensionsConfig }) => {
   //   label: 'Job handlers',
   //   component: <ConfigJobHandlers />
   // })
-  tabs.push({
-    label: 'Sharing',
-    component: <ConfigSharing />
-  })
   tabs.push({
```
Comment (magland) [inline src/containers/Config.tsx:41]: Switch the order of the config tabs in the config gui

File: src/containers/ConfigComputeResource.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# Compute resource configuration
```
Comment (magland) [inline src/containers/ConfigComputeResource.md:1]: The web gui now has instructions for configuring a remote compute resource

File: src/containers/ConfigComputeResource.md.gen.ts [hunk 5]
```diff
@@ -0,0 +1,3 @@
...\n|Using compute resource:|`{computeResourceUri}`|\n\n## Using a remote compute resource\n\nTo configure labbox to use a remote compute resource, create a `labbox_config.yml` file in the base directory of the labbox_ephys repo with the following content:\n\n```yaml\n---\ncompute_resource_uri: feed://...\njob_handlers:\n  local:\n    type: local\n  partition1:\n    type: remote\n  partition2:\n    type: remote\n  partition3:\n    type: remote\n  timeseries:\n    type: remote\n```\n\nreplacing `feed://...` with the feed of the compute resource (see the section below for information on hosting a compute resource). You will also need to give permission for this labbox server to access the compute resource. For this, you will need the kachery node ID for this labbox, which is given above.\n\n## Hosting a compute resource\n\nFollow these instructions for hosting the compute resource: [hosting a hither compute resource](https://github.com/flatironinstitute/hither/blob/master/doc/hosting_compute_resource.md)\n\nAs you run the configuration script, you will be prompted to give permission labbox-ephys to access this compute resource. You should provide the kachery node ID as provided above."
```
Comment (magland) [inline src/containers/ConfigComputeResource.md.gen.ts:1]: This is auto-generated

File: src/containers/ConfigComputeResource.tsx [hunk 6]
```diff
@@ -14,23 +18,21 @@ interface OwnProps {
...
     }
-
     return (
-        <div>
-            <h1>Compute resource configuration</h1>
-            <Content />
-        </div>
+        <Markdown
+            source={md}
+            substitute={substitute}
```
Comment (magland) [inline src/containers/ConfigComputeResource.tsx:29]: Convenience: pass substitute object into markdown (so we can use the markdown as a template file)

File: src/reducers/serverInfo.ts [hunk 7]
```diff
@@ -1,31 +1,49 @@
 import { Reducer } from 'react'
 import { SET_SERVER_INFO } from '../actions'
 
-interface ServerInfo {
-    nodeId: string | null
-    defaultFeedId: string | null
+type HandlerType = 'local' | 'remote'
+
+export interface ServerInfo {
```
Comment (magland) [inline src/reducers/serverInfo.ts:6]: We now receive the server info (including labbox config) in the browser

**Review by jsoules — APPROVED** [review]

**Pull request merged**